tutorial/README.md

```diff
@@ -341,6 +342,28 @@ db2 "INSERT INTO DB2INST1.CUSTOMERS(first_name, last_name, email) VALUES ('John'
 docker-compose -f docker-compose-db2.yaml down
 ```
 
+## Using Cassandra
+
+```shell 
+# Start the topology as defined in https://debezium.io/docs/tutorial/
+export DEBEZIUM_VERSION=1.3
+
+docker-compose -f docker-compose-cassandra.yaml up
+
+# Consume messages from a Debezium topic
+docker-compose -f docker-compose-cassandra.yaml exec kafka /kafka/bin/kafka-console-consumer.sh \
+    --bootstrap-server kafka:9092 \
+    --from-beginning \
+    --property print.key=true \
+    --topic server1.dbo.customers
+
+# Initialize and insert some test data 
+docker-compose -f docker-compose-cassandra.yaml exec cassandra bash -c 'cqlsh --file="/debezium/inventory.cql"'
```

Comment:
This can be done instratup script automatically, right after the server is started and initialized. No need for user to do it manually.